### DISCUSSION TITLE
Multiple selected blocks' actions (delete selected)

### DISCUSSION THREAD

**Discussion by cgvictor** [codex-team/editor.js#1942] [Q&A] [answered]
1. _Describe a problem._ Missing a feature to do something with multiple selected blocks inside an editor. Mainly, **the delete feature**. Open the https://editorjs.io/ main page, after that try to remove multiple blocks from sample content **without the need to 3-clicking on every block provided** (block menu click, delete click, confirm click).

2. Well, the "instant solution" would be to implement 'Del' button handler for this particular case, that deletes selected blocks, and that's it.
More sophisticated solution would be like: implement some API handlers' toolkit, that receives the list of selected blocks / selection context, and gives the ability to do (whatever) with selected blocks inside the handler code. And triggered with a hotkey, or any other suitable way.

3. _Are there any alternatives?_ For now, I had implemented a js hook in my application code, that does something like that:
`if(confirm('Clear all?')) [editor id].clear();`
But that's a pretty straightforward solution as I can presume.

**Comment by jeremyVignelles** [comment]
Related to #1626 ? I think it works with backspace.

**Comment by cgvictor** [comment, marked as answer]
> Related to #1626 ? I think it works with backspace.

Yep, `backspace` works, as an instant solution. Thank you for pointing on that hotkey, will use it for now
So, the feature question is now only 'do we plan any complex approach to multiple blocks' actions'
